### DISCUSSION TITLE
[question] support of official valve-mods like tfc, dod, etc?

### DISCUSSION THREAD

**Discussion by stamepicmorg** [theAsmodai/metamod-r#81] [Q&A] [answered]
**Just a question**. Can you add to the metamod-r support for official modifications that are available for download through the steamcmd and have official support of amxx? I [know](https://github.com/theAsmodai/metamod-r/pull/12) about custom loading a dll from config ini.

This mods succussfully works with rehlds (tested).

I'm just curious and I'm sorry... 😄 

### official valve mods:
[name]. [folder]. [dll]. [so]

Ricochet: ricochet, mp.dll, ricochet.so
Counter-Strike:Condition Zero, Deleted scenes: czeror, cz.dll,cz.so
DeathMatch Classic: dmc, dmc.dll, dmc.so
Day of Defeat: dod, dod.dll, dod.so
Team Fortress Classic: tfc, tfc.dll ,tfc.so

### ~3rd party:~
~Natural Selection: ns, ns.dll, ns_i386.so~
~The Specialists: ts, mp.dll, ts_i386.so, ts_i686.so~

**Comment by hajimura** [comment]
Why do you need this? Do specifically you hosts any servers besides Counter-Strike 1.6?

**Comment by hajimura** [comment]
ADD: CS:CZ is already supports metamod-r

**Comment by stamepicmorg** [comment, marked as answer]
@Phantomsq,
> Do specifically you hosts any servers besides Counter-Strike 1.6?

yes. hl1 servers, Ricochet (lol, why?), Natural Selection and The Specialists 3.0..

> Why do you need this? 

i dont. i can add path in my config manually. but original metamod supports this list of mods, and this metamod-r not. and i have this question.

> ADD: CS:CZ is already supports metamod-r

original Counter-Strike: Condition Zero != Counter-Strike:Condition Zero Deleted scenes

|  Condition Zero  | CZ Deleted scenes |
| ------------- | ------------- |
| czero  | czeror |
| cs.so | cz.so  |
| mp.dll | cz.dll  |

**Comment by stamepicmorg** [comment]
thanks!  ❤️
